rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -338,6 +338,12 @@ def test_exec(config):
                             raise TestExecError(
                                 "Command is throwing error while running bucket sync run"
                             )
+
+                    if config.bucket_sync_status:
+                        out = utils.bucket_sync_status(bucket.name)
+                        if out is False:
```

Comment:
Could we do `if not out` instead. The current statement evaluates False type not the boolean. Hence switching it helps.